Chinese/xinwang/xin.md

```diff
@@ -1,3 +1,3 @@
 小熊维尼又把头埋在罐子里闻了闻，说：“我记得有一个叔叔曾经告诉过我，说他看到奶酪过奶酪也是这种颜色的，那么也就是说这有可能是奶酪了喽？ 保险起见，还是让我先鉴定一下吧，恩，确实应该先鉴定一下。“
 
-小熊维尼吃饱了。
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.